vcsutils/webhookparser/bitbucketcloud.go

```diff
@@ -49,6 +49,10 @@ func (webhook *BitbucketCloudWebhook) parseIncomingWebhook(payload []byte) (*Web
 		return webhook.parsePrEvents(bitbucketCloudWebHook, vcsutils.PrCreated), nil
 	case "pullrequest:updated":
 		return webhook.parsePrEvents(bitbucketCloudWebHook, vcsutils.PrEdited), nil
+	case "pullrequest:fulfilled":
+		return webhook.parsePrEvents(bitbucketCloudWebHook, vcsutils.PrMerged), nil
+	case "pullrequest:rejected":
```

Comment:
Is "rejected" the only way a PR is closed in bitbucket cloud?

Reply:
To my knowledge, yes